apps/chat_server/lib/chat_server/contexts/starred_messages/delete_starred_message.ex

```diff
@@ -0,0 +1,19 @@
+defmodule ChatServer.DeleteStarredMessage do
+  require Logger
+
+  alias ChatServer.Schema
+
+  def call(message_id, user) do
+    message = Schema.Message.get(message_id)
+    starred_message = Schema.StarredMessage.find_by_message_and_user(message, user)
+    star_id = Map.get(starred_message, :id)
+
+    with {:ok, _record} <- Schema.StarredMessage.delete(starred_message) do
+        {:ok, %{id: star_id}}
```

Comment:
Same indentations here, looks like 4 but should be 2 deeper than `with` on line 11.